homeassistant/components/roborock/const.py

```diff
@@ -8,6 +9,44 @@
 CONF_BASE_URL = "base_url"
 CONF_USER_DATA = "user_data"
 
+# domain options
+CONF_INCLUDE_SHARED = "include_shared"
+DEFAULT_INCLUDE_SHARED = True
+
+# Option Flow steps
+SIZES = "sizes"
+DRAWABLES = "drawables"
+MAPS = "maps"
+
+DEFAULT_DRAWABLES = {
+    Drawable.CHARGER: True,
+    Drawable.CLEANED_AREA: False,
```

Comment:
Instead of adding all `Drawable` here, can we add only the ones that are enabled by default and looping over the `Drawable` enum in all places?

Reply:
I'm not fully following - sorry, can you expand?